### PR TITLE
fix(mod5 harris): fix bugs and improve perf with dichotomy

### PR DESCRIPTION
**Pull request by nicdub**
FIX 150

Needed to adapt cld_harris.py according to common harris_utils.py

### CODE REVIEW

File: backend/tests/test_modules.py [hunk 1]
```diff
@@ -79,7 +79,7 @@ def test_cld_harris() -> None:
     OUTPUT_CSV_FILE = os.path.join(OUTPUT_DIR, OUTPUT_CSV_FILENAME)
     print(f"INPUT_FILE={os.path.abspath(INPUT_FILE)}")
     print(f"OUTPUT_CSV_FILE={os.path.abspath(OUTPUT_CSV_FILE)}")
-    cld_harris.execute(INPUT_FILE, OUTPUT_CSV_FILE)
+    cld_harris.execute(INPUT_FILE, OUTPUT_CSV_FILE, 27.1, 27.7)
```
Comment (sbancal) [inline backend/tests/test_modules.py:82]: Surprised by constants hard written in a function call.
Good practice would be to declare it at the beginning of the file (or in a dedicated separate file) with a CAPITAL_NAME.

**Review by sbancal — COMMENTED** [review]
See suggestions about constants.
otherwise, LGTM

File: backend/tests/test_modules.py (hunk 1 above)
Comment (nicdub) [reply backend/tests/test_modules.py]: Thank you for your remarks, indeed as it was tests and I was changing the parameters to test the algo, I hadn't bothered to do things properly, it's corrected:)

**Review by sbancal — APPROVED** [review]
LGTM

**Review by sbancal — APPROVED** [review]
LGTM

**Comment by nicdub** [comment]
@blueur cld_harris.py and miner_harris.py have now more parameters (ucs and uts), I added default values for backend/ccfatigue/analyzer.py compatibility

**Pull request merged**